packages/application-components/src/components/page-content-containers/page-content-narrow/page-content-narrow.tsx

```diff
@@ -0,0 +1,37 @@
+import { ReactNode } from 'react';
+import styled from '@emotion/styled';
+
+export type TPageContentNarrow = {
+  children: ReactNode;
+};
+
+const LeftContentMarginColumn = styled.div`
+  grid-area: 'left-margin';
+`;
+
+const MainContentColumn = styled.div`
+  grid-area: 'main-content';
+`;
+
+const RightContentMarginColumn = styled.div`
+  grid-area: 'right-margin';
+`;
+
+const MainContainer = styled.section`
+  display: grid;
+  grid-template-areas: 'left-margin main-content right-margin';
+  grid-template-columns: 1fr minmax(400px, 600px) 1fr;
+  width: 100%;
+`;
+
+function PageContentNarrow(props: TPageContentNarrow) {
+  return (
+    <MainContainer>
+      <LeftContentMarginColumn />
+      <MainContentColumn>{props.children}</MainContentColumn>
+      <RightContentMarginColumn />
+    </MainContainer>
+  );
+}
```

Comment:
Nit: if grid cells are empty, we can use a `.` and there is no need to define elements

<img width="1795" alt="image" src="https://user-images.githubusercontent.com/1110551/209205511-ef5630b6-c465-437f-ad4c-30aa5c136517.png">

```suggestion
const Content = styled.div`
  grid-area: 'content';
`;

const Container = styled.section`
  display: grid;
  grid-template-areas: ". content .";
  grid-template-columns: 1fr minmax(400px, 600px) 1fr;
  width: 100%;
`;

function PageContentNarrow(props: TPageContentNarrow) {
  return (
    <Container>
      <Content>{props.children}</Content>
    </Container>
  );
}
```

Reply:
I didn't know about it. Thanks for the tip 🙇 

Updated in this commit: 5e306e1c1fabf0f61996604f1fb07d166f41e12e